examples/integrations/cdi/datasource-hikaricp-mysql/pom.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Don't forget -- now that the new year is here you'll need to update the copyright date.